blaze/server/tests/fixtures.py

```diff
@@ -0,0 +1,19 @@
+import pytest
```

Comment:
instead of creating a `fixtures.py`, you can create a `conftest.py` and `pytest` will automatically look for fixtures there. So, instead of having a function that makes a fixture, just define the fixture in `conftest.py` and use it wherever you want (it has to be from a file in the same directory where `conftest.py` is located).